data/mainnet/tokens.js

```diff
@@ -7185,4 +7218,207 @@ module.exports = {
     ],
     cmcRewardTokenSymbols: ['iFARM', 'DAI', 'WETH'],
   },
+  ape_BANANA_MATIC: {
```

Comment:
Put the new vaults up at the top, like with any new vaults. Below FARM and ETH entries.